schemas/npi_tables.sql

```diff
@@ -0,0 +1,43 @@
+DROP TABLE IF EXISTS meta.npi_variables;
+CREATE TABLE meta.npi_variables (
+    id serial primary key,
+    name text UNIQUE
+);
+
+COMMENT ON TABLE meta.npi_variables IS E'This table contains a list of variables and their corresponding identifiers used in `data.npi`. The variables include:
+
+* closing_of_public_venues
+* lockdown
+* non-essential_services_closure
+* religious_gatherings_banned
+* shelter_in_place
+* social_distancing
+* gathering_size_10_0
+* gathering_size_25_11
+* gathering_size_100_25
```

Comment:
Should this be `_10_25`?

Also see below when you insert the variables into the table